x-pack/filebeat/docs/inputs/input-netflow.asciidoc

```diff
@@ -73,6 +73,15 @@ The maximum number of packets that can be queued for processing.
 Use this setting to avoid packet-loss when dealing with occasional bursts
 of traffic.
 
+
+[float]
+[[workers]]
+==== `workers`
+
+The number of workers to read and decode concurrently netflow packets.
+Default is `1`. Note that in order to maximize the performance gains of multiple
+workers you also have to increase the number of output workers.
```

Comment:
Should we advise anything about the queue size or stay mum?

Reply:
hopefully soon enough we will become better equipped to provide more "precise" numbers but until then I did add here to switch to `throughput` preset. What are your thoughts? 🙂